### DISCUSSION TITLE
Automatically focus on title after creating a note

### DISCUSSION THREAD

**Discussion by kowalxx123** [standardnotes/forum#3440] [Suggestions]
When you create a note, the app currently moves keyboard focus to the note's content. Because most of my notes are just titles with no content, I would prefer if it first focused on the title. My ideal workflow would be: New note -> type title -> press Tab or Return -> type content (with no mouse clicking required).

Perhaps this can be an option in the settings? Thanks for considering.

**Comment by munkmode** [comment]
+1
